src/components/UserUpdate/UserUpdate.tsx

```diff
@@ -77,92 +83,196 @@ const UserUpdate: React.FC<UserUpdateProps> = ({ id }): JSX.Element => {
     }
   };
 
-  /* istanbul ignore next */
-  const cancelUpdate = () => {
-    window.location.reload();
-  };
-
   return (
     <>
-      <div id="userupdate" className={styles.userupdatediv}>
-        <form>
-          {/* <h3 className={styles.settingstitle}>Update Your Details</h3> */}
-          <div className={styles.dispflex}>
-            <div>
-              <label>{t('firstName')}</label>
-              <input
-                type="input"
-                id="firstname"
-                placeholder={t('firstName')}
-                autoComplete="off"
-                required
-                value={formState.firstName}
-                onChange={(e) => {
-                  setFormState({
-                    ...formState,
-                    firstName: e.target.value,
-                  });
-                }}
-              />
-            </div>
-          </div>
-          <div className={styles.dispflex}>
-            <div>
-              <label>{t('lastName')}</label>
-              <input
-                type="input"
-                id="lastname"
-                placeholder={t('lastName')}
-                autoComplete="off"
-                required
-                value={formState.lastName}
-                onChange={(e) => {
-                  setFormState({
-                    ...formState,
-                    lastName: e.target.value,
-                  });
-                }}
-              />
-            </div>
-          </div>
-          <div className={styles.dispflex}>
-            <div>
-              <label>{t('email')}</label>
-              <input
-                type="email"
-                id="email"
-                placeholder={t('email')}
-                autoComplete="off"
-                required
-                value={formState.email}
-                onChange={(e) => {
-                  setFormState({
-                    ...formState,
-                    email: e.target.value,
-                  });
-                }}
-              />
+      <div
+        className="modal fade"
+        id="userUpdateModal"
+        tabIndex={-1}
+        aria-hidden="true"
+      >
+        <div className="modal-dialog modal-dialog-scrollable">
+          <div className="modal-content">
+            <div className="modal-header">
+              <h5 className="modal-title" id="notificationModalLabel">
+                {t('updateYourDetails')}
+              </h5>
+              <button
+                type="button"
+                className="close"
+                data-dismiss="modal"
+                aria-label="Close"
+              >
+                <span aria-hidden="true">&times;</span>
+              </button>
```

Comment:
New fields on user update page are redundant. There is no input for them inside the TypeDefs and resolvers in API.